### PR TITLE
Task4 Pull Request

### PR DESCRIPTION
**Pull request by davidmshou**

### CODE REVIEW

File: students/DavidShoubridge/session02/series.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    else:
+        series = [x, y]
+        for i in range(n - 1):
+            series.append(series[i] + series[i + 1])
+        return series[len(series) - 1]
+
+
+if __name__ == "__main__":
+    """
+        The first 3 assert statements test whether the fibonacci function \
```
Comment (BFriedland) [inline students/DavidShoubridge/session02/series.py]: The end-of-line escape character doesn't need to be used for triple-quote strings.

File: students/DavidShoubridge/session02/series.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    assert(fibonacci(7)) == 13
+
+    assert(lucas(0)) == 2
+    assert(lucas(3)) == 4
+    assert(lucas(7)) == 29
+
+    assert(sum_series(2)) == 1  # Testing for return from Fibonacci series.
+    assert(sum_series(2, 2, 1)) == 3  # Testing for return from Lucas series.
+    assert(sum_series(2, 4, 3)) == 7  # Testing for return from x and y series.
+    assert(sum_series(6, 2, 1)) == 18  # Testing for return from Lucas series.
```
Comment (BFriedland) [inline students/DavidShoubridge/session02/series.py]: Though this isn't a stylistic error for grading purposes, it's considered a best practice to include a new line at the end of the file. This supports some historical unix utilities.

File: students/DavidShoubridge/session02/series.py (hunk 1 above)
Comment (davidmshou) [inline students/DavidShoubridge/session02/series.py]: Oh it doesn't? That's good to know, my linter keeps telling me to even in
my big block quotes.

Thanks!

File: students/DavidShoubridge/session02/series.py (hunk 2 above)
Comment (ppham) [inline students/DavidShoubridge/session02/series.py]: This line doesn't do what you are expecting.

assert(fibonacci(1)) will always be true as long as you return a non-zero number, or a non-None value. assert is a statement, and doesn't return a value (it is not an expression.)

assert(fibonacci(1)) == 1 will execute silently, and not do the comparison that you want.

instead, you should use:

`assert(fibonacci(1) == 1)`

**Pull request merged**